### PR TITLE
Initial version of the WebSocket Next client ADR

### PR DESCRIPTION
**Pull request by cescoffier**

### CODE REVIEW

**Comment by cescoffier** [comment]
@mkouba Finally wrote the few missing lines!

**Review by geoand — APPROVED** [review]

File: adr/0003-websocket-next-client.adoc [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        .baseUri(uri)
+        .pathParam("name", "Roxanne")
+        .connectAndAwait();
+// <3> Use the connection to send messages if needed (the client endpoint can also retrieve the connection)
+connection.sendTextAndAwait("Hi!");
+----
+
+In this example, the connector is injected in a bean. The connector is used to create a new connection. The connection is configured with the base URI and path parameters. The connection is then established by calling the `connectAndAwait` method (asynchronous methods are also available). The connection can be used to send messages, close the connection, etc.
+
+The duality between the _client endpoint_ and the _connector_ separates the configuration and establishment of the websocket connection from the logic. The _client endpoint_ is used to define the logic of the WebSocket client, while the _connector_ is used to configure and create the connection.
```
Comment (mkouba) [inline adr/0003-websocket-next-client.adoc:103]: Should we also mention the `BasicWebSocketConnector` that can be used without a client endpoint class? 

https://github.com/quarkusio/quarkus/blob/main/extensions/websockets-next/deployment/src/test/java/io/quarkus/websockets/next/test/client/BasicConnectorTest.java

Comment (cescoffier) [reply adr/0003-websocket-next-client.adoc:103]: Ah yes! Let me add this!

Comment (cescoffier) [reply adr/0003-websocket-next-client.adoc:103]: Added as well as a note on the broadcast and the possibility to list the active connections.

**Review by mkouba — APPROVED** [review]

**Pull request merged**